spec/facility_spec.rb

```diff
@@ -2,25 +2,192 @@
+      vin: '123456789abcdefgh', 
+      year: 2012, 
+      make: 'Chevrolet', 
+      model: 'Cruz', 
+      engine: :ice
+    })
+    @bolt = Vehicle.new({
+      vin: '987654321abcdefgh', 
+      year: 2019, 
+      make: 'Chevrolet', 
+      model: 'Bolt', 
+      engine: :ev
+    })
+    @camaro = Vehicle.new({
+      vin: '1a2b3c4d5e6f', 
+      year: 1969, 
+      make: 'Chevrolet', 
+      model: 'Camaro', 
+      engine: :ice
+    })
+    @registrant_1 = Registrant.new('Bruce', 18, true)
+    @registrant_2 = Registrant.new('Penny', 16)
+    @registrant_3 = Registrant.new('Tucker', 15)
   end
+
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('Albany DMV Office')
-      expect(@facility.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
-      expect(@facility.phone).to eq('541-967-2014')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('Albany DMV Office')
+      expect(@facility_1.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
+      expect(@facility_1.phone).to eq('541-967-2014')
+      expect(@facility_1.services).to eq([])
+      expect(@facility_1.daily_hours).to eq('9-5')
+      expect(@facility_1.holidays).to eq('All of them')
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+  describe 'registration' do 
+    it 'registers vehicles' do 
+      @facility_1.add_service('Vehicle Registration')
+      expect(@cruz.registration_date).to be nil
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
+      #cruz
+      @facility_1.register_vehicle(@cruz)
+      expect(@cruz.registration_date).to be_a Date
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      expect(@facility_1.collected_fees).to eq(100)
+      #camaro
+      @facility_1.register_vehicle(@camaro)
+      expect(@camaro.registration_date).to be_a Date
+      expect(@camaro.plate_type).to eq(:antique)
+      #bolt
+      @facility_1.register_vehicle(@bolt)
+      expect(@bolt.registration_date).to be_a Date
+      expect(@bolt.plate_type).to eq(:ev)
```

Comment:
Ok, I can see why you tested all three vehicles since they're each creating a different plate type.